src/libs/auth.ts

```diff
@@ -520,7 +520,19 @@ authStore.subscribe(
 // extending Window interface to access Appcues
 declare global {
   interface Window {
-    Appcues: any;
+    Appcues?: {
+      /** Identifies the current user with an ID and an optional set of properties. */
+      identify: (userId: string, properties?: any) => void;
+      /** Notifies the SDK that the state of the application has changed. */
+      page: () => void;
+      /** Forces specific Appcues content to appear for the current user by passing in the ID. */
+      show: (contentId: string) => void;
+      /** Fire the callback function when the given event is triggered by the SDK */
+      on: ((eventName: Exclude<string, 'all'>, callbackFn: (event: any) => void | Promise<void>) => void) &
+        ((eventName: 'all', callbackFn: (eventName: string, event: any) => void | Promise<void>) => void);
+      /** Clears all known information about the current user in this session */
+      reset: () => void;
```

Comment:
Not including the `track` method here since events should instead be sent to AppCues via Bard. Thank you for raising @nawatts!

Reply:
... or maybe not. Reverting for now and putting the PR on hold until security conversation is resolved.